core/controllers/editor.py

```diff
@@ -242,7 +242,7 @@ def get(self, exploration_id):
                 get_value_generators_js()),
             'title': exploration.title,
             'visualizations_html': jinja2.utils.Markup(visualizations_html),
-            'ALL_LANGUAGE_CODES': feconf.ALL_LANGUAGE_CODES,
+            'ALL_LANGUAGE_CODES': constants.ALL_LANGUAGE_CODES,
```

Comment:
Again, as constants are already provided to the frontend, you don’t need to send them through here.